server/php/UploadHandler.php

```diff
@@ -132,18 +132,13 @@ public function __construct($options = null, $initialize = true, $error_messages
             'identify_bin' => 'identify',
             'image_versions' => array(
                 // The empty image version key defines options for the original image:
-                // Keep in mind that these options are inherited by all other image versions!
+                // Warning: these image manipulations are inherited by all other image versions! 
+                // Keep in mind that this does not apply to 'no_cache'.
```

Comment:
"Warning" is too strong a word for what this does.
If "Warning" is removed I'm fine with the change though.